### ISSUE TITLE
Proxy anchor loss

### ISSUE THREAD

**Issue by geonm** [KevinMusgrave/pytorch-metric-learning#32]
### Proxy Anchor Loss for Deep Metric Learning
arxiv: https://arxiv.org/abs/2003.13911

<img src="https://user-images.githubusercontent.com/21214529/78543534-be530b00-7833-11ea-959c-bce6bd66c5f9.png" width="300">

I implemented the proxy anchor loss of `Proxy Anchor Loss for Deep Metric Learning` for **_pytorch, mxnet, and tensorflow_**.

This is not an official code! but it works well.

### CARS 196 results
![image](https://user-images.githubusercontent.com/21214529/78564082-e6a03100-7856-11ea-949a-54e73323ed72.png)

+ red one => proxy anchor loss
+ blue one => norm softmax

### codes
Pytorch version is here: https://github.com/geonm/proxy-anchor-loss/blob/master/pytorch-proxy-anchor.py

Can you add the proxy anchor loss to this project?

**Comment by KevinMusgrave** [comment]
Thanks, this looks great! I'll include it in the next dev version

**Comment by tjddus9597** [comment]
Hi, I am sungyeon kim, author of Proxy Anchor Loss for Deep Metric Learning. 
We also uploaded the code for Proxy Anchor loss. When including to next versions, it would be good to implement Proxy-Anchor Loss by referring to both geonm's code and our code.
Our repository is here: https://github.com/tjddus9597/Proxy-Anchor-CVPR2020

**Comment by KevinMusgrave** [comment]
@geonm @tjddus9597 Thanks for your code! I borrowed a bit from both, and it's available as ProxyAnchorLoss in v0.9.82.
